wordle-web/layouts/default.vue

```diff
@@ -15,13 +17,39 @@
     <v-footer app>
       <span>&copy; {{ new Date().getFullYear() }}</span>
     </v-footer>
+
+    <v-navigation-drawer
+    v-model="drawer"
```

Comment:
To get this to appear, you'll need to change the boolean value of `drawer` to true and back to false when you wish to close it. You can do this with a `v-btn`

You'll also need to declare it as a variable in your typescript.